dutch/over-het-hek/sleutel/nacht/deze-nacht.md

```diff
@@ -1 +1,4 @@
 Plotseling komen vage beelden naar boven ...
+
+Je beseft dat je droomt, alsof je in een slecht verhaal zit, en ziet dat alles verkeerd gespeld is. Een boom heet plotseling booom en benzinetankstation heet bedritanktontiats. Maar zoals in alle dromen die van zichzelf bewust zijn heb je de keuze om [wakker te worden](../welkom.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.